src/main/java/com/google/sps/servlets/FetchQuestionsServlet.java

```diff
@@ -45,9 +45,9 @@
    */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    List<QuestionObject> questions = new ArrayList<>();
+    List<Question> questions = new ArrayList<>();
     
-    String query;
+    String query = Utility.fetchQuestionsQuery;
```

Comment:
Still don't get why you create the query in the Utility file. I think the query should be in the file it is used, so the reader can know what it's doing.

Reply:
because it is going to be used by Andres in another servlet, so it is better to have it be reusable, and the reader can easily go to the Utility file to check what that does, same with all of the constants

Reply:
Ok, it occurs to me then, that instead of having these in the Utility file, all queries could go on another Java class, in order to not clutter this one too much. Totally optional, though.

Reply:
mhmmm so maybe like a SqlQueries.java? sounds good, but for the moment I will leave it as a TODO to avoid merge conflicts, thanks!